### DISCUSSION TITLE
有几个问题想请教您

### DISCUSSION THREAD

**Discussion by Arthur-Sun** [courses-at-nju-by-hfwei/discrete-math-problem-sets#13] [Q&A] [answered]
老师，我有几个问题想请教您一下，第一个问题是
![2E8817E2-0685-485B-AF82-59859F318443](https://user-images.githubusercontent.com/79981763/123379996-74358100-d5c1-11eb-8d89-8fefaa417d85.png)图一圈起来的这种表示为什么是合理的，这样的话
｛c，d｝似乎没人管啊。第二个是图二的题目的答案，为什么A的幂集的大小为2的n次方就可以推出结论啊？第三个是图三的这种表示，为什么是正确的啊？相似的一个是您录的最后一节课把｛1，2，3｝上的恒等置换写作（2，3）（3，2），这样不就表示2到3，3到2，2到2了嘛
![6B2B5474-BC37-4409-BE00-E864221C2427](https://user-images.githubusercontent.com/79981763/123380420-0178d580-d5c2-11eb-9987-624c60363fd7.png)

![C33807BF-FD5B-4207-BFB6-2908CC8837C1](https://user-images.githubusercontent.com/79981763/123380305-dd1cf900-d5c1-11eb-8e4a-029312dd8d08.jpeg)

**Comment by hengxin** [comment, marked as answer]
@Arthur-Sun (1) 这就是用逻辑里的分配律展开的。 (2) 2^n 是有穷数。无穷多个两两不相交的集合相交, 元素个数无穷。(3) 验算一下就可以了，使用我录制的讲 permutation group 中介绍的方法，本质上是验证两个函数相等。恒等置换可以写成 (2 3) (3 2) (或者 (2 3) (2 3))，直观理解就是先把 2、3 调换位置，然后再调换一次，就相当于没有调换。要验算的话就是, (2 3) (3 2) 作用在 1 上得到 1，作用在 2 上得到 2, 作用在 3 上得到 3，因此是恒等置换。

**Reply by Arthur-Sun** [reply]
嗯嗯，谢谢老师，第一个和第三个我懂了，但是第二个，这样展开之后，我圈起来的这部分是不是一定为假啊，不然的话就能推出｛a,b｝＝｛a｝，似乎不成立啊。
